src/main/java/org/wikivoyage/listings/input/DumpDownloader.java

```diff
@@ -47,10 +49,53 @@ public void downloadDumpFromUrl(String dumpUrl, String dumpFilename) throws IOEx
             IOUtils.closeQuietly(in);
         }
 
+        Collections.sort(availableDumps);
+        Collections.reverse(availableDumps);
+        
+        /*
+         * I check the JSON status file of the last dump, if the dump is "partial" (in-progress)
+         * I discard it in favour of the previous one, if exist
+         */
+        if (availableDumps.size()>0){
+        		String dumpStatusURL = indexUrl + availableDumps.get(0) + "/dumpstatus.json"; 
+    			String dumpStatus = this.getDumpStatus(dumpStatusURL);
+        		if (this.isPartialDump(dumpStatus)) {
+        			availableDumps.remove(0);
+        		}
+        	}
+        
         return availableDumps;
     }
 
-    public String dumpUrl(String language, String dumpId)
+    public String getDumpStatus(String dumpStatusURL) throws IOException {
+    		String dumpStatus="";
+    		InputStream in = new URL(dumpStatusURL).openStream();
+        try {
+        		dumpStatus = IOUtils.toString(in);
+        } finally {
+            IOUtils.closeQuietly(in);
+        }
+        return dumpStatus;
+    }
+
+    /*
```

Comment:
Javadoc comment shoud begin with `/**`.
Would you mind adding what the method does? For instance `Check whether a dump is partial (currently being generated and not complete) or not`.
Thanks!

Reply:
Something like this is ok?
/**
     * Check whether a dump is partial (currently being generated and not complete) or not
     * Data dumps/Status format
     * https://meta.wikimedia.org/wiki/Data_dumps/Status_format
     * @param	content of dumpstatus.json     
     * @return	true if the dump is partial	 
     */`

Reply:
Yes perfect! Just maybe make `Data dumps/Status format` into `Data dumps/Status format:` so that we know it is the description of the content at that URL.

Reply:
ok, great!